ocs_ci/ocs/acm/acm.py

```diff
@@ -0,0 +1,229 @@
+import logging
+import time
+
+from selenium.webdriver.support.wait import WebDriverWait
+from selenium.webdriver.support import expected_conditions as ec
+
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.framework import MultiClusterConfig
+from ocs_ci.utility.utils import TimeoutSampler
+from ocs_ci.ocs.ui.acm_ui import AcmPageNavigator
+from ocs_ci.ocs.ui.views import locators
+from ocs_ci.ocs.ui.base_ui import login_ui
+
+
+log = logging.getLogger(__name__)
+
+
+class AcmAddClusters(AcmPageNavigator):
+    """
+    ACM Page Navigator Class
+
+    """
+
+    def __init__(self, driver):
+        super().__init__(driver)
+        self.page_nav = locators[self.ocp_version]["acm_page"]
+
+    def import_cluster_ui(self, cluster_name, kubeconfig_location):
+        """
+
+        Args:
+            cluster_name (str): cluster name to import
+            kubeconfig_location (str): kubeconfig file location of imported cluster
+
+        """
+        self.navigate_clusters_page()
+        self.do_click(self.page_nav["Import_cluster"])
+        self.do_send_keys(
+            self.page_nav["Import_cluster_enter_name"], text=f"{cluster_name}"
+        )
+        self.do_click(self.page_nav["Import_mode"])
+        self.do_click(self.page_nav["choose_kubeconfig"])
+        log.info(f"Coping Kubeconfig {kubeconfig_location}")
+        kubeconfig_to_import = copy_kubeconfig(kubeconfig_location)
+        log.info(kubeconfig_to_import)
+        self.do_click(self.page_nav["Kubeconfig_text"])
+        for line in kubeconfig_to_import:
+            self.do_send_keys(self.page_nav["Kubeconfig_text"], text=f"{line}")
+            time.sleep(2)
+        log.info(f"Submitting import of {cluster_name}")
+        self.do_click(self.page_nav["Submit_import"])
+
+    def import_cluster(self, cluster_name, kubeconfig_location):
+        """
+        Import cluster using UI
+
+        Args:
+            cluster_name:
+            kubeconfig_location:
+
+        Returns:
+
+        """
+
+        self.import_cluster_ui(
+            cluster_name=cluster_name, kubeconfig_location=kubeconfig_location
+        )
+        for sample in TimeoutSampler(
+            timeout=450,
+            sleep=60,
+            func=validate_cluster_import,
+            cluster_name=cluster_name,
+        ):
+            if sample:
+                log.info(f"Cluster: {cluster_name} successfully imported")
+            else:
+                log.error(f"import of cluster: {cluster_name} failed")
+
+
+def copy_kubeconfig(file):
+    """
+
+    Args:
+        file: (str): kubeconfig file location
+
+    Returns:
+        list: with kubeconfig lines
+
+    """
+
+    try:
+        with open(file, "r") as f:
+            txt = f.readlines()
+            return txt
+
+    except FileNotFoundError:
+        log.error("file not found")
```

Comment:
you need to handle this error, or you will try to use `None` as kubeconfig during the import on Line 47